static/js/containers/PostPage.js

```diff
@@ -49,14 +51,18 @@ type PostPageProps = {
   commentInFlight: boolean,
   // from the router match
   channelName: string,
-  postID: string
+  postID: string,
+  showDialog: boolean,
```

Comment:
Should this be refactored this to a name that's a little more specific to which dialog it is for?

Reply:
yeah that's a good idea